templates/vue/src/store/actions.js

```diff
@@ -303,6 +303,17 @@ export async function updateUserFavourites({ commit, dispatch }, favourites) {
   }
 }
 
+// avatars
+
+export async function updateAvatar({ commit, dispatch }, newAvatar) {
```

Comment:
Maybe it's best this is called `updateUserSettings` and avatar is just a property of that. This will make it flexible enough to add future user settings easily using the same structure.

Reply:
The retrieving of the avatar has remained the same, but the saving of the avatar has been modified to use the userSettings object which has the avatar property in it.